com.unity.shadergraph/Editor/Generation/Enumerations/InterpolationModifier.cs

```diff
@@ -0,0 +1,12 @@
+namespace UnityEditor.ShaderGraph
+{
+    [GenerationAPI]
+    internal enum InterpolationModifier
+    {
+        Linear,
```

Comment:
I am wondering if change related to modifier may conflict with this PR here: https://github.com/Unity-Technologies/Graphics/pull/3468/files#diff-13f5ff3fe703f5b644e6dd0281cd393c6987650f8cefa0d1f4e596332e95ebb6R105

Reply:
Hm, indeed it probably would. The question will be if that PR should make use of this PR's introduction of a formal Interpolation Modifier to the API, or just wait for that PR to merge and then I will merge with it and untangle the conflicts